iree_tests/conftest.py

```diff
@@ -246,26 +248,29 @@ def collect(self):
         #     ...
 
         test_directory = self.path.parent
+        relative_test_directory = test_directory.relative_to(IREE_TESTS_ROOT).as_posix()
```

Comment:
Ah, need to update https://github.com/nod-ai/SHARK-TestSuite/blob/main/iree_tests/update_config_xfails.py also

Reply:
Fix: https://github.com/nod-ai/SHARK-TestSuite/pull/280